presto-geospatial/pom.xml

```diff
@@ -62,6 +62,12 @@
         </dependency>
```

Comment:
I do not see tests like `TestLogicalPlanner` for queries which could benefit from this rule and no new tests in ATQ

Reply:
@kokosing Grzegorz, this one is tricky. I cannot extend TestLogicalPlanner and AbstractTestQueries tests with spatial join tests because these tests live in presto-main and presto-tests neither of which can depend on presto-geospatial without introducing circular references. Without presto-geospatial, we cannot use geospatial functions in the queries.  Also, both tests rely on TPCH dataset which doesn't have any geospatial data. Do you have any suggestions on how to work around these issues?

Reply:
@kokosing I cannot add tests which use geospatial functions to presto-main. If I just move this test to presto-main it will compile successfully, but then fail with an error: Function ST_Contains not registered. To fix that error I'd need to modify the query runner to install GeoPlugin which comes from presto-geospatial. If I try that I'll end up with circular references between modules.

Reply:
See, I did not had any issues with dependencies. I have just followed your way of having tests in `presto-geospatial`

Reply:
https://gist.github.com/kokosing/d25bdf2f1e04efc5dc67774c6f4b39f1